packages/core/src/components/toast/toaster.tsx

```diff
@@ -66,11 +66,13 @@ export interface IToasterProps extends IProps {
     inline?: boolean;
 
     /**
-     * Position of `Toaster` within its container. Note that `LEFT` and `RIGHT` are disallowed
-     * because Toaster only supports the top and bottom edges.
+     * Position of `Toaster` within its container.
+     *
+     * Note that only `TOP` and `BOTTOM` are supported because Toaster only
+     * supports the top and bottom edge positioning.
      * @default Position.TOP
      */
-    position?: Position;
+    position?: Position.TOP | Position.BOTTOM;
```

Comment:
this is not complete. it's actually TOP_* and BOTTOM_*: can align left/right too.